components/Ripple/src/MDCStatefulRippleView.m

```diff
@@ -171,7 +181,7 @@ - (void)setRippleHighlighted:(BOOL)rippleHighlighted {
     // If ripple becomes highlighted we initiate a ripple with animation.
     [self updateRippleColor];
     [self beginRippleTouchDownAtPoint:_lastTouch animated:_didReceiveTouch completion:nil];
-  } else {
+  } else if (!rippleHighlighted) {
```

Comment:
Is this necessary for this change? It seems like an independent bug that could be fixed in a separate PR.